src/main/scala/tilelink/Parameters.scala

```diff
@@ -534,48 +537,81 @@ class TLSlavePortParameters private(
   // Does this Port manage this ID/address?
   def containsSafe(address: UInt) = findSafe(address).reduce(_ || _)
 
-  private def supportHelper(
+  private def addressHelper(
       safe:    Boolean,
-      member:  TLManagerParameters => TransferSizes,
+      member:  TLSlaveParameters => TransferSizes,
```

Comment:
`// member filters out the sizes being checked based on the opcode being emitted or supported`